SQL/vetclinicData.sql

```diff
@@ -0,0 +1,18 @@
+BEGIN TRANSACTION;
+INSERT INTO "clients" ("client_id","first_name","last_name","phone","email") VALUES (1,'john','doe','555-321-6543','johndoe@mail.com');
+INSERT INTO "pets" ("pet_id","name","species","gender","dob","owner") VALUES (1,'Isis','cat','FS','2006-11-01',1);
+INSERT INTO "appointments" ("date","time","client","pet","reason") VALUES ('2020-05-03','2020-05-03 11:00:00',1,1,'Annual Exam');
+INSERT INTO "users" ("user_id","username","password") VALUES (1,'bob','5e884898da28047151d0e56f8dc6292773603d0d6aabbdd62a11ef721d1542d8');
+COMMIT;
+
+/*

```

Comment:
The below lines can be deleted as they are also present above, though uncommented.

Reply:
The lines below are not the same as the lines above. The lines commented below show how to actually assign the foreign key values in SQL syntax. The lines above have the data hard-coded from being exported from the database. When we want to insert new data we would use the commented lines below. However, the lines above are able to be imported into a database if needed. They both serve a purpose.